vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -376,7 +376,7 @@ - (void)publisherActivityFromURL:(NSURL *)URL
 {
   auto visitData = [self visitDataForURL:URL tabId:0];
   visitData.favicon_url = std::string(faviconURL.absoluteString.UTF8String);
-  ledger->GetPublisherActivityFromUrl(1, visitData, std::string(publisherBlob.UTF8String));
+  ledger->GetPublisherActivityFromUrl(1, visitData.Clone(), std::string(publisherBlob.UTF8String));
```

Comment:
do we need to clone here? Can't we just std::move?

Reply:
The returned value is a not a Ptr variant so nothing to move. I have to go back through the code and replace all the base type and Clone() combinations with Ptr and move's instead

Reply:
I see, this is the raw struct and you're converting it back to Ptr